Main/Include/game.h

```diff
@@ -206,6 +206,13 @@ class game
   static cchar* Insult();
   static truth TruthQuestion(cfestring&, int = 0, int = 0);
   static void DrawEverything();
+  static void UpdateShowItemsAtPlayerPos(bool bAllowed);
+  static void UpdateAltSilhouette(bool bAllowed);
+  static v2 CalculateStretchedBufferCoordinatesFromDungeonSquarePos(v2 v2SqrPos);
+  static int IntemUnderCode(int iCycleValue);
```

Comment:
The code appears to be consistent, but you wrote `IntemUnderCode`. Did you mean `ItemUnderCode`?

Reply:
xD